### PR TITLE
implement teal colour palette RSC-362

### PR DESCRIPTION
**Pull request by glenhunter**
https://issues.sonatype.org/browse/RSC-362

The main colour used in these swatches was the teal for the focus borders. I created a variable for that, otherwise there were very few of these colours used in the code.

### CODE REVIEW

File: lib/src/base-styles/_nx-radio-checkbox.scss [hunk 1]
```diff
@@ -37,8 +37,8 @@
 
   &:focus-within {
     .nx-radio-checkbox__control {
-      border-color: #02a6d9;
-      stroke: #02a6d9;
+      border-color: $nx-focus-border-color;
+      stroke: $nx-focus-border-color;
       filter: drop-shadow(0 0 3px rgba(2, 166, 217, 0.4));
```
Comment (rpokorny) [inline lib/src/base-styles/_nx-radio-checkbox.scss:42]: It's not written in hex format but I believe this is the same color.

File: lib/src/scss-shared/_nx-colors.scss [hunk 2]
```diff
@@ -17,7 +17,7 @@ $nx-sonatype-orange: #f89100;
 // Tints based on HSL lightness value
 $nx-color-blue-41: #3A4D95;
 $nx-color-blue-42: #2B66AB;
-$nx-color-blue-44: #00A1E2;
+$nx-color-blue-44:$nx-teal-400;
```
Comment (rpokorny) [inline lib/src/scss-shared/_nx-colors.scss:20]:
```suggestion
$nx-color-blue-44: $nx-teal-400;
```

**Review by rpokorny — APPROVED** [review]
+1 aside from the missed rgba value

File: lib/src/base-styles/_nx-radio-checkbox.scss (hunk 1 above)
Comment (glenhunter) [reply lib/src/base-styles/_nx-radio-checkbox.scss]: > It's not written in hex format but I believe this is the same color.

https://github.com/sonatype/sonatype-react-shared-components/pull/241/commits/3e0e378d3293dcee864b59be5b66b959b2725845

**Pull request merged**